src/components/manga/reader/Page.tsx

```diff
@@ -105,7 +120,7 @@ const Page = React.forwardRef((props: IProps, ref: any) => {
     }, [handleVerticalScroll]);
 
     return (
-        <div ref={ref} style={{ margin: '0 auto' }}>
+        <div ref={ref} style={{ margin: 'auto' }}>
```

Comment:
what's the effect of this change?

Reply:
it more or less aligns the image to the center instead of being at top shown in the BEFORE portrait mode

Reply:
**Single**
![ltr](https://user-images.githubusercontent.com/61667570/137553444-fd2d0d1f-cf6d-4870-b53e-10d69c686b07.png)


**Webtoon**
![webtoon](https://user-images.githubusercontent.com/61667570/137553484-a086c47a-0714-45b9-aeb7-e9c792053e16.png)


**Continuous**
![continous](https://user-images.githubusercontent.com/61667570/137553573-db48ed4e-1ac8-498d-9403-d4556b461b5f.png)


**Computed for continuous and webtoon**
![computed](https://user-images.githubusercontent.com/61667570/137553925-d12942fa-0026-46a6-ab90-3b8df7394356.png)


**Computed for single**
![computed single](https://user-images.githubusercontent.com/61667570/137554111-52033a28-4938-4f5a-9192-e7263660bd36.png)


I'm getting expected results when running it with Chrome/electron, center for single page and no margin for webtoon/continuous.